src/components/ContextBar.js

```diff
@@ -16,7 +16,10 @@ const contextBarStyles = {
     padding: '1rem',
     color: colors.accent(),
     backgroundColor: colors.accent(0.2)
-  }
+  },
+  //original hover for reference
+  // hover: { boxShadow: `inset -6px 0px ${colors.accent()}` }
```

Comment:
this needs another UX review, figured I'd leave this here for a before/after

Reply:
I talked with Joy on this, we were going for the complementary color of the top tabbar with consistent hover coloring. Both have +0.2 accent on the base color for their hover effect